lib/livebook/runtime/erl_dist/runtime_server.ex

```diff
@@ -497,6 +505,12 @@ defmodule Livebook.Runtime.ErlDist.RuntimeServer do
     {:noreply, state}
   end
 
+  def handle_cast(:disable_dependencies_cache, state) do
+    System.put_env("MIX_INSTALL_FORCE", "true")
```

Comment:
Should we allow `MIX_` prefixed env vars to be created by Settings env vars? I tested and it force all the time to be recompiled.

Reply:
You mean if we should disallow env vars like that? I think there's no reason to do that, and people may actually want to customize `MIX_INSTALL_DIR` :)